src/Core/src/Handlers/View/ViewHandlerOfT.Tizen.cs

```diff
@@ -0,0 +1,264 @@
+using System;
+using Tizen.UIExtensions.Common;
+using ElmSharp;
+
+using Rectangle = Microsoft.Maui.Graphics.Rectangle;
+using Size = Microsoft.Maui.Graphics.Size;
+using Point = Microsoft.Maui.Graphics.Point;
+using ESize = ElmSharp.Size;
+using ERect = ElmSharp.Rect;
+
+namespace Microsoft.Maui.Handlers
+{
+	public abstract partial class ViewHandler<TVirtualView, TNativeView> : INativeViewHandler
+	{
+		bool _disposedValue;
+
+		EvasObject? INativeViewHandler.NativeView => WrappedNativeView;
+		EvasObject? INativeViewHandler.ContainerView => ContainerView;
+
+		protected new EvasObject? WrappedNativeView => (EvasObject?)base.WrappedNativeView;
+
+		public new WrapperView? ContainerView
+		{
+			get => (WrapperView?)base.ContainerView;
+			protected set => base.ContainerView = value;
+		}
+
+		public void SetParent(INativeViewHandler parent) => Parent = parent;
+
+		public CoreUIAppContext? Context => MauiContext?.Context;
+
+		public INativeViewHandler? Parent { get; private set; }
+
+		public EvasObject? NativeParent => Context?.BaseLayout;
+
+		~ViewHandler()
+		{
+			Dispose(disposing: false);
+		}
+
+		public override void NativeArrange(Rectangle frame)
+		{
+			var nativeView = WrappedNativeView;
+
+			if (nativeView == null)
+				return;
+
+			if (frame.Width < 0 || frame.Height < 0)
+			{
+				// This is just some initial Forms value nonsense, nothing is actually laying out yet
+				return;
+			}
+
+			if (NativeParent == null)
+				return;
+
+			var updatedGeometry = new Rectangle(ComputeAbsolutePoint(frame), new Size(frame.Width, frame.Height)).ToEFLPixel();
+
+			if (nativeView.Geometry != updatedGeometry)
+			{
+				nativeView.Geometry = updatedGeometry;
+			}
```

Comment:
NUI와 공통으로 정의된 Extensions method 사용하기 (`using Tizen.UIExtensions.ElmSharp;`)
```suggestion
			nativeView.UpdateBounds(new Rectangle(ComputeAbsolutePoint(frame), new Size(frame.Width, frame.Height)).ToPixel());
```